x-pack/plugins/security_solution_serverless/server/cloud_security/cloud_security_metering.ts

```diff
@@ -27,17 +33,25 @@ export const cloudSecurityMetringCallback = async ({
   try {
     const cloudSecurityUsageRecords: UsageRecord[] = [];
 
-    const cspmUsageRecord = await getCspmUsageRecord({
-      esClient,
-      projectId,
-      logger,
-      taskId,
-      lastSuccessfulReport,
-      abortController,
-    });
-
-    if (cspmUsageRecord) {
-      cloudSecurityUsageRecords.push(cspmUsageRecord);
+    const postureTypes: PostureType[] = [
+      CSPM_POLICY_TEMPLATE,
+      KSPM_POLICY_TEMPLATE,
+      CNVM_POLICY_TEMPLATE,
+    ];
+
+    for (const postureType of postureTypes) {
+      const usageRecord = await getCloudSecurityUsageRecord({
```

Comment:
nit: consider using promise all

Reply:
added it to https://github.com/elastic/security-team/issues/7256